### PR TITLE
Fix title

### PR DESCRIPTION
**Pull request by VaibhavPage**
/assign @magaldima 
/kind feature
/sig contributor-experience

### CODE REVIEW

**Comment by magaldima** [comment]
please fix merge conflict

File: seattle/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# KubeCon Seattle 2018 New K8s Contributor Workshop
+# KubeCon Seattle 2018 - New Kubernetes Contributor Workshop
```
Comment (magaldima) [inline seattle/README.md]:
```suggestion
# KubeCon Seattle 2018: New Kubernetes Contributor Workshop
```

**Comment by miles-** [comment]
:hourglass_flowing_sand:

**Comment by magaldima** [comment]
fixes #157

**Comment by magaldima** [comment]
closes #157

**Pull request merged**